atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetrics.java

```diff
@@ -205,32 +204,44 @@ private MetricsForStrategy(
             sweepDelayMetric = new CurrentValueMetric<>();
 
             TargetedSweepMetricPublicationFilter filter = createMetricPublicationFilter();
+
+            registerProgressMetricsFilter(strategy, filter);
             registerProgressMetrics(strategy, filter);
             outcomeMetrics = SweepOutcomeMetrics.registerTargeted(manager, tag, filter);
         }
 
-        private void registerProgressMetrics(String strategy, TargetedSweepMetricPublicationFilter filter) {
-            // This is kind of against the point of metrics-filter, but is needed for our filtering
```

Comment:
nit: maybe let's keep this comment since this still does look _weird_ to someone reading this without context